config/default.yml

```diff
@@ -0,0 +1,9 @@
+AllCops:
+  Include:
+    - '**/test/**/*'
+
+Minitest/AssertNil:
+  Description: 'Check if your test uses `assert_nil` instead of `assert_equal(nil, something)`.'
+  StyleGuide: 'https://github.com/rubocop-hq/minitest-style-guide#assert-nil'
+  Enabled: true
+  VersionAdded: '0.1.0'
```

Comment:
I have one (probably last) feedback for this PR. The other day my comment was wrong 💦  `VersionAdded` does not include tiny version. Can you update to `VersionAdded: '0.1'`?

Reply:
hey @koic. I did update it.